src/Parser/Rogue/Outlaw/Modules/RogueCore/ComboPoints.js

```diff
@@ -16,7 +17,7 @@ class ComboPoints extends Analyzer {
   
 
   makeExtraSuggestion(spell) {
-    return <Wrapper>Avoid wasting combo points when casting <SpellLink id={spell.id}  />. </Wrapper>;
+    return <Wrapper>Avoid wasting combo points when casting <SpellLink id={spell.id}  />. Note that some combo point wastage is unavoidable due to second saber slash procs during the duration of <SpellLink id={193356}  />.  </Wrapper>;
```

Comment:
It's prefered that you use `SPELLS.BROADSIDES.id` instead of hardcoding the spell id. This makes it both easier to read the code and avoids having the spell id hardcoded in multiple different places.